samples/subsys/tracing/sample.yaml

```diff
@@ -8,10 +8,11 @@ common:
     regex:
       - "thread_a: Hello World from (.*)!"
       - "thread_b: Hello World from (.*)!"
+    fixture: gpio_loopback
 tests:
   sample.tracing.user:
     extra_args: CONF_FILE="prj_user.conf"
-    integration_platforms:
+    platform_allow:
```

Comment:
why do you need to change these testcases?

Reply:
@ycsin - does this look ok now?